modules/ROOT/pages/workday/workday-connector-design-center.adoc

```diff
@@ -1,63 +1,67 @@
-= Workday Design Center Configuration - Mule 4
+= Workday - Design Center Configuration - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
-
 :page-aliases: workday-design-center.adoc
 
-The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a Design environment. For more information, see the Flow Designer Tour.
+Anypoint Design Center's Flow Designer enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
 
 To create an app in Flow Designer:
 
 * Configure the input source (trigger) for your app.
-* Add the connector as a component to process the input for the app.
+* Configure the target component for your app.
 
 For information on Design Center field values, see
 the xref:workday/workday-reference.adoc[Workday Connector Reference].
 
-
 == Configure the Input Source Trigger
 
+A trigger starts your app. Workday Connector provides the *New Object Trigger* operation that starts your app when a new object is created in Workday. Alternatively, you can use the HTTP Listener operation to start your app from a browser 
```

Comment:
No need to boldface the name of the operation; init. cap. is sufficient.
Bold is for direct reference to UI labels.